src/isNotNil.js

```diff
@@ -0,0 +1,6 @@
+const { complement, isNil } = require('ramda')
```

Comment:
It's a shame we don't have crocks in funky. Crocks `isNil` accounts for NaN which is nice.
```
//Crocks
const { not, isNil} = require('crocks')

const isNotNil = not(isNil)

isNotNil(NaN) // false
```

```
//Crocks
const { complement, isNil} = require('ramda')

const isNotNil = complement(isNil)

isNotNil(NaN) // true
```